v23.1/architecture/sql-layer.md

```diff
@@ -152,7 +152,7 @@ For more information, including examples and limitations, see [Online Schema Cha
 
```

Comment:
Using the terminology from the SQL layer to update the Transaction Layer request/operation confusion. Namely, switching to requests, which aligned with the Concurrency Manager section and our other Arch docs. Open to changes, if this isn't technically correct.